modules/utils/src/merkle.ts

```diff
@@ -15,7 +16,9 @@ export const generateMerkleTreeData = (transfers: CoreTransferState[]): { root:
   });
 
   // Generate tree
-  const tree = new MerkleTree(leaves, keccak256, { sortPairs: true });
+  const tree = new merkle.Tree();
```

Comment:
This isn't well documented in `wasm_bindgen`, but you'll need to call `free` on the tree or this will leak memory.